rfc006-release-planning-mgt.md

```diff
@@ -0,0 +1,110 @@
+1. The Release Manager will track progress towards the release's goals as reported in ZenHub through regular iterations (sprints), and will be responsible for identifying slippage and risk. The Release Manager will provide regular updates to the project's members as part of the regular meetings, although overall progress will be available anytime from ZenHub for members with access.
+1. If the release appears at risk for slippage, the Release Manager will coordinate with the Working Groups to see if progress can be brought back to the baseline, or if items will need to be removed from the release. The Release Manager will inform the PSC if the latter occurs, to determine if this is feasible, and if not how it should be handled.
+1. Working Groups will provide an update/demo of the results of their work in preparation for the release. Once the PSC and other stakeholders are satisfied that it meeds the commitment, members of the project can perform the necessary reviews/approvals, merge pull requests, and package the Working Groups' deliverables.
+
+
+## Toolset
+
+In as described above, this process will leverage three major tools to plan and manage releases.
+
+- Wiki: The high-level goals and commitments of a Release should be documented on a publicly-available wiki, to align expectations and provide transparency. The Release Manager will collect proposals from the Working Groups, and facilitate the discussions between Working Groups and the PSC to reach a consensus on the Release's contents.
+- GitHub: Work towards the goals of the Release will be represented by Issues in GitHub. GitHub will also be used to distribute the end products, either directly from source repositories, or as packages (e.g. .ZIP files) for end users to download from the **Releases** page of the appropriate repository.
+- ZenHub: ZenHub will provide organization and visibility through its ability to group individual work items (e.g. Issues) into higher-level Epics, and show progress toward their completion.
+
+
+## In Scope
+
+All Working Groups and sub-projects of The Good Docs Project are eligible for inclusion in a Release. That said, the precise contents of any Release should be reached through consensus between the PSC, who should seek to maximize the value delivered, and the Working Groups, who should ensure commitments are feasible.
+
+
+## Out of Scope
+
+The following items are out of scope for this proposal, as they're either not integral to a Release Planning & Management plan, or are addressed by other (potentially upcoming) RFCs:
+
+- Versioning: RFC-008 will detail the versioning or numbering standard for the project.

```

Comment:
I feel that our versioning strategy should be a prerequisite for this step, and we should be referencing it from this RFC.